superchain/superchain.go

```diff
@@ -76,55 +78,57 @@ const (
 )
 
 type ChainConfig struct {
-	Name         string `yaml:"name"`
-	ChainID      uint64 `yaml:"chain_id"`
-	PublicRPC    string `yaml:"public_rpc"`
-	SequencerRPC string `yaml:"sequencer_rpc"`
-	Explorer     string `yaml:"explorer"`
+	Name         string `yaml:"name" toml:"name"`
+	ChainID      uint64 `yaml:"chain_id" toml:"chain_id"`
+	PublicRPC    string `yaml:"public_rpc" toml:"public_rpc"`
+	SequencerRPC string `yaml:"sequencer_rpc" toml:"sequencer_rpc"`
+	Explorer     string `yaml:"explorer" toml:"explorer"`
 
-	SuperchainLevel SuperchainLevel `yaml:"superchain_level"`
+	SuperchainLevel SuperchainLevel `yaml:"superchain_level" toml:"superchain_level"`
 
 	// If StandardChainCandidate is true, standard chain validation checks will
 	// run on this chain even if it is a frontier chain.
-	StandardChainCandidate bool `yaml:"standard_chain_candidate,omitempty"`
+	StandardChainCandidate bool `yaml:"standard_chain_candidate,omitempty" toml:"standard_chain_candidate,omitempty"`
 
 	// If SuperchainTime is set, hardforks times after SuperchainTime
 	// will be inherited from the superchain-wide config.
-	SuperchainTime *uint64 `yaml:"superchain_time"`
+	SuperchainTime *uint64 `yaml:"superchain_time" toml:"superchain_time"`
 
-	BatchInboxAddr Address `yaml:"batch_inbox_addr"`
-
-	Genesis ChainGenesis `yaml:"genesis"`
+	BatchInboxAddr Address `yaml:"batch_inbox_addr" toml:"batch_inbox_addr"`
 
 	// Superchain is a simple string to identify the superchain.
 	// This is implied by directory structure, and not encoded in the config file itself.
-	Superchain string `yaml:"-"`
+	Superchain string `yaml:"-" toml:"-"`
 	// Chain is a simple string to identify the chain, within its superchain context.
 	// This matches the resource filename, it is not encoded in the config file itself.
-	Chain string `yaml:"-"`
+	Chain string `yaml:"-" toml:"-"`
 
 	// Hardfork Configuration Overrides
-	HardForkConfiguration `yaml:",inline"`
+	HardForkConfiguration `yaml:",inline" toml:",inline"`
+
+	BlockTime           uint64 `yaml:"block_time" toml:"block_time"`
+	SequencerWindowSize uint64 `yaml:"seq_window_size" toml:"seq_window_size"`
+
+	Genesis ChainGenesis `yaml:"genesis" toml:"genesis"`
 
-	BlockTime           uint64 `yaml:"block_time"`
-	SequencerWindowSize uint64 `yaml:"seq_window_size"`
+	Addresses AddressList `toml:"addresses"`
 
 	// Optional feature
-	Plasma *PlasmaConfig `yaml:"plasma,omitempty"`
+	Plasma *PlasmaConfig `yaml:"plasma,omitempty" toml:"plasma,omitempty"`
```

Comment:
See other comment. I think we should place this above `Genesis`.

Reply:
https://github.com/ethereum-optimism/superchain-registry/pull/386/commits/1f362c8813018ed8f50efdd0017a08c91d38bb22